astropy/constants/iau2015.py

```diff
@@ -7,7 +7,7 @@
 import numpy as np
 
 from .constant import Constant
-from .codata2014 import G
```

Comment:
Was this not removed from #8517 on purpose?

Reply:
Since we only use `G` perhaps it is OK to do `from .config.codata import G`? (Though the replacement you have seems fine too.)

Reply:
@mhvk , actually, I tried `from .config.codata import G` first but Python complains that `codata` is not a module and threw error.

Reply:
I think it was an oversight in #8517 to have not modified this line. The new code is good and correct as far as I can tell.